app/src/main/java/com/locoquest/app/dao/IBenchmarkDAO.kt

```diff
@@ -2,10 +2,6 @@ package com.locoquest.app.dao
 
 import com.locoquest.app.dto.Benchmark
 import retrofit2.Call
-import retrofit2.Callback
```

Comment:
Removed unused imports to avoid creating dependencies. You want to avoid these as much as possible to reduce code vulnerabilities! 

https://stackoverflow.com/questions/979057/any-reason-to-clean-up-unused-imports-in-java-other-than-reducing-clutter#:~:text=An%20unused%20import%20still%20creates,related%20to%20that%20module%2C%20etc.

Reply:
The IDE can (and should) organize imports automatically.  Do that, instead of manually merging a branch.
It will make no difference in runtime performance, as imports are optimized by the compiler at compile time.